### ISSUE TITLE
some problems in experiments

### ISSUE THREAD

**Issue by ykxian** [OpenFogStack/celestial-videoconferencing-evaluation#3]
Hi @pfandzelter I encountered some problems after repeating the experiment several times.
1. 
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/6d770ba5-7216-4250-8b35-557c03009ff5)
I performed the following operations on the host with Docker installed, and now it seems to be running normally.
```
sudo apt-get purge docker-ce docker-ce-cli containerd.io docker-compose-plugin
sudo apt-get autoremove
sudo iptables -F
sudo iptables -t nat -F
sudo systemctl restart NetworkManager
sudo systemctl restart systemd-networkd
sudo rm -rf /var/lib/docker
sudo rm -rf /etc/docker
sudo rm -rf /usr/libexec/docker
sudo rm -rf /var/run/docker.sock
sudo groupdel docker
```
I'm not sure which commands were effective or if they would be helpful to you.

2. The number of packets received is less than expected.
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/7ab6d7bd-a819-4479-ac0f-23aa6b60400d)
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/8d4685e3-84e1-4868-88a7-f59f16437b67)
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/7867a9cf-e9a9-49f8-8e37-3916653b17f5)

3. The actual latency is very high. After stabilizing, the latency is approximately 4 seconds.
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/105ff646-7343-447b-9462-2237669cd2ef)
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/a5211d6b-34b8-4f7f-942e-e6877fd35386)
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/8fe9f305-ebd7-4ea3-b75b-99bb88464069)

4. When interrupted, this code does not work, causing the subsequent code not to execute.
onewaymeasurement.py
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/6d66ef9d-5fff-4141-b032-89501b5eeac4)

I am not sure if questions 2 and 3 are related to my equipment or network environment. Even if I don't use the host with Docker installed, the experiment results are still the same.
Thank you!

**Comment by pfandzelter** [comment]
Hi again, thanks for continuing to try this. I found a bug in Celestial in an unrelated experiment that caused the bandwidth to be only 1/1000th of what it should be. Maybe it's related to this. I will try it out on my setup

**Comment by ykxian** [comment]
Indeed, it is related to this. I increased the bandwidth by a thousand times, and the results improved significantly.

**Comment by ykxian** [comment]
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/709de674-ee02-477e-86a8-390e46d227f7)
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/9aa864c6-5630-47bb-bec2-d636da80bb84)

**Comment by pfandzelter** [comment]
That looks much better! I have merged the fixes for Celestial and will shortly update this repository. I have made minor changes to the HTTP API to now reflect the units of information, e.g. `delay_us`. Not having units caused the previous bug...

**Comment by ykxian** [comment]
Thanks again! It works well now. The latency fluctuation is somewhat large.
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/9e930c1b-0ae0-47b6-8d89-e3d59e9e0184)
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/5d79f221-a726-4314-86f8-13ae0bedd8ce)

**Comment by ykxian** [comment]
Sometimes there is negative latency.
![image](https://github.com/OpenFogStack/celestial-videoconferencing-evaluation/assets/150787122/3f6ff4e7-71d6-457c-a846-6c2d797cf154)

**Comment by ykxian** [comment]
I synchronized the time on three servers, and now it's working well.
